app/javascript/schools/courses/components/CourseCertificates__Root.re

```diff
@@ -0,0 +1,260 @@
+                                          ),
+                                        "issued_count",
+                                      )
+                                      ->str}
+                                   </p>
+                                 </div>
+                                 {Certificate.active(certificate)
+                                    ? <div
+                                        className="flex flex-wrap text-gray-600 font-semibold text-xs mt-1">
+                                        <span
+                                          className="px-2 py-1 border rounded bg-secondary-100 text-primary-600 mt-1 mr-1">
+                                          {t("auto_issue_tag")->str}
+                                        </span>
+                                      </div>
+                                    : React.null}
+                               </div>
+                               <div className="flex">
+                                 <a
+                                   title=editTitle
+                                   className="w-10 text-sm text-gray-700 hover:text-gray-900 cursor-pointer flex items-center justify-center hover:bg-gray-200"
+                                   onClick={_ =>
+                                     send(
+                                       OpenEditCertificateDrawer(certificate),
+                                     )
+                                   }>
+                                   <i className="fas fa-edit" />
+                                 </a>
+                                 {if (Certificate.issuedCertificates(
+                                        certificate,
+                                      )
+                                      == 0) {
+                                    let title =
+                                      t(
+                                        ~variables=[|
+                                          (
+                                            "name",
+                                            Certificate.name(certificate),
+                                          ),
+                                        |],
+                                        "delete_button_title",
+                                      );
+
+                                    <a
+                                      title
+                                      className="w-10 text-sm text-gray-700 hover:text-gray-900 cursor-pointer flex items-center justify-center hover:bg-gray-200"
+                                      onClick={_event =>
+                                        WindowUtils.confirm(
+                                          "Are you sure you want to delete this certificate?",
+                                          () =>
+                                          deleteCertificate(certificate, send)
+                                        )
```

Comment:
Can we use `<ConfirmWindow/>` component instead of the browser dialog?

Reply:
@vinutv Are we switching to the custom confirmation component for _all_ use-cases? 🤔 

I didn't use the component here, because it has a _lot_ of mandatory parameters: `(~title, ~body, ~confirmButtonText,
 ~cancelButtonText, ~onConfirm, ~onCancel, ~disableConfirm)`. I wasn't even sure what some of the params were for. For example: `~disableConfirm`.

Reply:
I though we were switching away from browser dialogs as we have pretty good alternative

Reply:
@bodhish I've spoken to @vinutv about this and we've decided to continue using browser dialogs for simple confirmations. This appears to be the standard for most modern web-apps.

The custom `ConfirmWindow` components was built with the purpose of displaying _complex_ confirmations that would usually include some kind of user input. The original idea was something akin to Github's _"type in the name of this repo to confirm its deletion"_ popup. However, as it stands, the `ConfirmWindow` component doesn't actually support that kind of usage (it doesn't accept `children`) - it'll need to be upgraded to allow for complex use.